Library/Formula/android-ndk.rb

```diff
@@ -19,9 +16,6 @@ def install
     if MacOS.prefer_64_bit?
```

Comment:
Same here. Drop the `if` and corresponding `end` and then leave the `x86_64` chmod and system call in place.